src/components/timeline/TrackCustomMarkerGraph.js

```diff
@@ -152,18 +151,17 @@ class TrackCustomMarkerCanvas extends React.PureComponent<CanvasProps> {
 
         const getX = (time) =>
           Math.round((time - rangeStart) * millisecondWidth);
+        // For line graphs, ensure y is at least half the stroke's line width
+        // so that it won't be cut off the bottom edge of the graph.
+        const minY = type === 'bar' ? 0 : deviceLineWidth * 0.5;
         const getY = (i) => {
           const unitValue = _calculateUnitValue(
             type,
             minNumber,
             maxNumber,
             samples[i]
           );
-          // Add on half the stroke's line width so that it won't be cut
-          // off the edge of the graph.
-          return Math.round(
-            deviceHeight - deviceHeight * unitValue - deviceLineHalfWidth
-          );
+          return Math.floor(deviceHeight - deviceHeight * unitValue - minY);
```

Comment:
So we do `Math.floor` now because we don't want to make heights lke `deviceHeight - 0.2` invisible, right? I think this changes the shape of the graph a bit, but that's fine I think.